URLs.md

```diff
@@ -116,6 +116,16 @@ git checkout -b admin internal/admin
 For browsing `internal/admin` branch on web:
 https://cgit.freebsd.org/${repo}/log/?h=internal/admin
 
+For pushing, either specify the full refspec:
+```
+git push origin HEAD:refs/internal/admin
+```
+
+Or setting `push.default` gitconfig to use the `upstream`:
```

Comment:
s/upstream/freebsd/ since that's what we're trying to use everywhere

Reply:
`upstream` here is the config value of `push.default`, to ask git push to where the branch came from:
https://git-scm.com/docs/git-config#Documentation/git-config.txt-pushdefault

Maybe change the wording like "setting `push.default` gitconfig to use `upstream` model"?

Reply:
Yes, `upstream` is the hardcoded value that git uses to represent the upstream branch from which the current working branch is created.

Reply:
I'd probably amend this to something like:

Or setting `push.default` to `upstream` which will make `git push` to push the current branch back to its upstream by default, which is more suitable for our workflow: